catalog-info.yaml

```
@@ -235,3 +252,47 @@ spec:
          access_level: MANAGE_BUILD_AND_READ
         everyone:
           access_level: READ_ONLY
+
+---
+##
+# buildkite-pipeline-eui-release
+# EUI Release - handle (pre)release process on merge to main
+##
+
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/e57ee3bed7a6f73077a3f55a38e76e40ec87a7cf/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: buildkite-pipeline-eui-release
+  description: EUI pipeline to release the latest version of @elastic/eui
+  links: [
+    {
+      title: "EUI - release",
+      url: "https://buildkite.com/elastic/eui-release",
+    }
+  ]
+
+spec:
+  type: buildkite-pipeline
+  owner: group:eui-team
+  system: buildkite
+  implementation:
+    apiVersion: buildkite.elastic.dev/v1
+    kind: Pipeline
+    metadata:
+      name: eui-release
+    spec:
+      repository: elastic/eui
+      pipeline_file: ".buildkite/pipelines/pipeline_release.yml"
+      provider_settings:
+        build_branches: true
+        build_tags: false
+        build_pull_requests: false
+        filter_enabled: true
+        # TODO: remove the test-automatic-releases condition when the pipeline is production-ready
+        filter_condition: build.branch == "main" || build.branch == "build/test-automatic-releases"
```

If we're still testing, shouldn't we be merging this PR into this test branch instead of to main? or am I misunderstanding the purpose of this PR

---

Any changes to `catalog-info.yml` have to be in `main` to be picked up. There's a task that looks for new changes periodically and only looks at that one branch. I had to find that out the hard way. 😬

---

ahh right, I always forget that.

Shouldn't this line omit the `build.branch` for now in that case, to prevent it from running on main itself?

```suggestion
        filter_condition: build.branch == "build/test-automatic-releases"
```

---

Yes, I think it sh|could ignore `main`. That way it only runs against the test branch for now, and when it's ready, the `catalog-info` be updated to point at main and test for real.

---

Gotcha, as long as we're consciously making the decision to yolo (and respond promptly in case anything goes awry)! None of my comments are blocking, so feel free to address them or merge as you see fit